ibis/impala/compiler.py

```diff
@@ -1043,16 +1043,16 @@ def _sign(translator, expr):
 
 _operation_registry = {
```

Comment:
You're removing those operations from Impala, so you need to add them back by extending the `operation_registry` you created in `base_sql`.